_hardware/controllers.md

```diff
@@ -753,14 +753,14 @@ Marlin supports a wide variety of display controllers, from simple character-bas
     <iframe class="youtube rj" width="300" height="225" src="https://youtube.com/embed/{{ videoid }}"></iframe>
   {% endfor %}
   {% for img in cont.images %}
-    {% if img.front %}<a href="{{ fldr }}{{ img.front }}_front.jpg" target="_blank" title="{{ img.alt }} Front"><img src="{{ fldr }}{{ img.front }}_front_thumb.jpg" alt="{{ img.alt }} Front"></a>{% endif %}
+    {% if img.front %}<img class="myImages" id="myImg" src="{{ fldr }}{{ img.front }}_front.jpg" alt="{{ img.alt }} Front" style="width:100%;max-width:300px">{% endif %}
```

Comment:
This minor, but you'll want to update these "my" classes/ids to something more descriptive/specific to what they are for.